recipes/qtmockwebserver/all/patches/0004-fixed-server-hangs.patch

```diff
@@ -0,0 +1,120 @@
+From 8e3b9b71282267c2e869484107b3a39bb9a055e0 Mon Sep 17 00:00:00 2001
```

Comment:
Thanks @tomas-krupa 

What is the provenance of this patch? Please note that while we consider patches to build scripts to ensure a correct build, we do not regularl consider patches on C or C++ code (https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#policy-on-patches)  - this is for the QtMockwebserver maintainers to consider.

Reply:
you are right, the patch has been removed. thanks!